engine/execution/messages.go

```diff
@@ -17,18 +18,18 @@ type ComputationOrder struct {
 }
 
 type ComputationResult struct {
-	ExecutableBlock    *entity.ExecutableBlock
-	StateSnapshots     []*delta.SpockSnapshot
-	StateCommitments   []flow.StateCommitment
-	Proofs             [][]byte
-	Events             []flow.EventsList
-	EventsHashes       []flow.Identifier
-	ServiceEvents      flow.EventsList
-	TransactionResults []flow.TransactionResult
-	ComputationUsed    uint64
-	StateReads         uint64
-	TrieUpdates        []*ledger.TrieUpdate
-	ExecutionDataID    flow.Identifier
+	ExecutableBlock         *entity.ExecutableBlock
+	StateSnapshots          []*delta.SpockSnapshot
+	StateCommitments        []flow.StateCommitment
+	Proofs                  [][]byte
+	Events                  []flow.EventsList
+	EventsHashes            []flow.Identifier
+	ServiceEvents           flow.EventsList
+	TransactionResults      []flow.TransactionResult
+	ComputationUsed         uint64
+	StateReads              uint64
+	TrieUpdates             []*ledger.TrieUpdate
+	ExecutionDataProvideJob *provider.ProvideJob
```

Comment:
It's because the storing of the blobs happens asynchronously. I can split the done channel out separately, but at some point somewhere we will still need to check the error and log it.